python/pyarrow/tests/test_parquet.py

```diff
@@ -3015,6 +3015,21 @@ def test_dictionary_array_automatically_read():
     assert result.schema.metadata is None
 
 
+@pytest.mark.pandas
+def test_categorical_roundtrip():
+    # ARROW-5480, this was enabled by ARROW-3246
+    from io import BytesIO
+    df = pd.DataFrame({'x': pd.Categorical(['a', 'a', 'b', 'b'])})
+
+    buf = BytesIO()
+    df.to_parquet(buf)
+
+    # This reads back object, but I expected category
```

Comment:
this comment reads a bit unclear to me (it now reads back categorical?)

Reply:
Sorry I copy-pasted this from the bug report